hq/app/AppComponents.scala

```diff
@@ -24,13 +28,37 @@ class AppComponents(context: Context)
     csrfFilter,
     new HstsFilter()
   )
+  implicit val awsClient: AWSSimpleSystemsManagement = AWSSimpleSystemsManagementFactory("eu-west-1", "security")
+
+  val configraun: Configuration = {
+
+    configuration.getOptional[String]("stage") match {
+      case Some("DEV") =>
+        val stack = configuration.get[String]("stack")
+        val app = configuration.get[String]("app")
+        val stage = "DEV"
+        Configraun.loadConfig(Identifier(Stack(stack), App(app), Stage.fromString(stage).get)) match {
+          case Left(a) =>
+            Logger.error(s"Unable to load Configraun configuration from AWS (${a.message})")
+            sys.exit(1)
```

Comment:
O_O

Why are we calling sys.exit in a webserver?!

Reply:
Fast fail in the event of bad config. Perhaps a comment would be useful, or there may be a better (more Play) answer, but the action is correct IMO.